From f4a00b5cbd898763c8d72da6eb1a4cb3583812cf Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Fri, 7 Jul 2023 06:33:34 +0000
Subject: [PATCH] Update the kernel config and clang verion

This will update the clang version to 17.0.2

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/kernel/AndroidBoard.mk                             | 2 +-
 .../gmin64/config-lts/lts2021-chromium/x86_64_defconfig   | 8 ++++----
 .../gmin64/config-lts/lts2022-chromium/x86_64_defconfig   | 8 ++++----
 3 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index 8999494..48dd19d 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -22,7 +22,7 @@ kernel: $(PRODUCT_OUT)/kernel
 
 else
 
-TARGET_KERNEL_CLANG_VERSION := r487747
+TARGET_KERNEL_CLANG_VERSION := r487747c
 CLANG_PREBUILTS_PATH := $(abspath $(INTEL_PATH_DEVICE)/../../../prebuilts/clang)
 
 ifneq ($(TARGET_KERNEL_CLANG_VERSION),)
diff --git a/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
index 40eefdb..db10143 100644
--- a/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
@@ -2,15 +2,15 @@
 # Automatically generated file; DO NOT EDIT.
 # Linux/x86_64 5.15.119 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="Android (9796371, based on r487747) clang version 17.0.0 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362)"
+CONFIG_CC_VERSION_TEXT="Android (10087095, +pgo, +bolt, +lto, -mlgo, based on r487747c) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362)"
 CONFIG_GCC_VERSION=0
 CONFIG_CC_IS_CLANG=y
-CONFIG_CLANG_VERSION=170000
+CONFIG_CLANG_VERSION=170002
 CONFIG_AS_IS_LLVM=y
-CONFIG_AS_VERSION=170000
+CONFIG_AS_VERSION=170002
 CONFIG_LD_VERSION=0
 CONFIG_LD_IS_LLD=y
-CONFIG_LLD_VERSION=170000
+CONFIG_LLD_VERSION=170002
 CONFIG_CC_HAS_ASM_GOTO=y
 CONFIG_CC_HAS_ASM_GOTO_OUTPUT=y
 CONFIG_TOOLS_SUPPORT_RELR=y
diff --git a/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig
index 0268a87..8a3da96 100644
--- a/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2022-chromium/x86_64_defconfig
@@ -2,15 +2,15 @@
 # Automatically generated file; DO NOT EDIT.
 # Linux/x86_64 6.1.34 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="Android (9796371, based on r487747) clang version 17.0.0 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362)"
+CONFIG_CC_VERSION_TEXT="Android (10087095, +pgo, +bolt, +lto, -mlgo, based on r487747c) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362)"
 CONFIG_GCC_VERSION=0
 CONFIG_CC_IS_CLANG=y
-CONFIG_CLANG_VERSION=170000
+CONFIG_CLANG_VERSION=170002
 CONFIG_AS_IS_LLVM=y
-CONFIG_AS_VERSION=170000
+CONFIG_AS_VERSION=170002
 CONFIG_LD_VERSION=0
 CONFIG_LD_IS_LLD=y
-CONFIG_LLD_VERSION=170000
+CONFIG_LLD_VERSION=170002
 CONFIG_CC_HAS_ASM_GOTO_OUTPUT=y
 CONFIG_CC_HAS_ASM_GOTO_TIED_OUTPUT=y
 CONFIG_TOOLS_SUPPORT_RELR=y
-- 
2.17.1

